front/app/containers/IdeasShow/components/RightColumnDesktop/index.tsx

```diff
@@ -99,25 +93,28 @@ const RightColumnDesktop = ({
                 <Divider />
               </>
             )}
-            {/* Participation method is 'ideation' */}
-            {showIdeaReactionControl && (
-              <>
-                <ReactionControl styleType="shadow" ideaId={ideaId} size="4" />
-                <Divider />
-              </>
-            )}
-            {/* Participation method is 'voting' */}
-            {/* TODO: Fix this the next time the file is edited. */}
-            {/* eslint-disable-next-line @typescript-eslint/no-unnecessary-condition */}
-            {phase && ideaIsInParticipationContext && votingConfig && (
-              <Box pb="24px" mb="24px" borderBottom="solid 1px #ccc">
-                {votingConfig.getIdeaPageVoteInput({
-                  ideaId,
-                  phase,
-                  compact: false,
-                })}
-              </Box>
-            )}
+            {participationMethod === 'ideation' &&
+              showIdeationReactions(idea.data) && (
+                <>
+                  <ReactionControl
+                    styleType="shadow"
+                    ideaId={ideaId}
+                    size="4"
+                  />
+                  <Divider />
+                </>
+              )}
+            {participationMethod === 'voting' &&
```

Comment:
Already covered by the votingConfig check (which we only have if PM is `voting`) but this is more explicit and aligns it with the other PM options.